src/app/components/TransactionDialog/dialog.module.scss

```diff
@@ -0,0 +1,63 @@
+.dialog {
+  @apply tw-relative
+    tw-font-body
+    tw-bg-black tw-border-0 tw-rounded-3xl tw-text-sov-white;
+
+  padding: 30px 0 50px 0 !important;
+  max-width: 468px;
+}
+
+.dialog h1 {
+  @apply tw-text-center tw-text-3xl tw-leading-tight tw-font-semibold tw-normal-case;
+  margin-bottom: 30px;
+}
+
+.dialog button[data-close] {
+  width: 42px;
```

Comment:
You have some properties (`border`, `position`) here that have standard tailwind values so you could use `@apply tw-...`.